MockHealth/src/main/java/com/openlattice/integrations/mockHealth/DemoHealth.java

```diff
@@ -0,0 +1,259 @@
+package com.openlattice.integrations.mockHealth;
+
+import com.dataloom.authorization.PermissionsApi;
+import com.dataloom.client.RetrofitFactory;
+import com.dataloom.client.RetrofitFactory.Environment;
+import com.dataloom.data.serializers.FullQualifedNameJacksonDeserializer;
+import com.dataloom.edm.EdmApi;
+import com.dataloom.mappers.ObjectMappers;
+import com.kryptnostic.rhizome.configuration.service.ConfigurationService;
+import com.openlattice.shuttle.Flight;
+import com.openlattice.shuttle.MissionControl;
+import com.openlattice.shuttle.Shuttle;
+import com.openlattice.shuttle.dates.DateTimeHelper;
+import com.openlattice.shuttle.edm.RequiredEdmElements;
+import com.openlattice.shuttle.edm.RequiredEdmElementsManager;
+import com.openlattice.shuttle.util.Parsers;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import org.joda.time.DateTimeZone;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import retrofit2.Retrofit;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DemoHealth {
+    private static final Logger logger = LoggerFactory
+            .getLogger(DemoHealth.class);
+    private static final Environment environment = Environment.STAGING;
+
+    private static final DateTimeHelper dateHelper0 = new DateTimeHelper( DateTimeZone.forOffsetHours( -8 ), "MM/dd/yyyy" );
+
+    public static void main(String[] args) throws InterruptedException {
+
+        FullQualifedNameJacksonDeserializer.registerWithMapper(ObjectMappers.getYamlMapper());
+        FullQualifedNameJacksonDeserializer.registerWithMapper(ObjectMappers.getJsonMapper());
+
+        final String healthPath = args[0];
+        final String jwtToken = args[1];
+
+        final SparkSession sparkSession = MissionControl.getSparkSession();
+
+        logger.info("Using the following idToken: Bearer {}", jwtToken);
+
+        Retrofit retrofit = RetrofitFactory.newClient( environment, () -> jwtToken );
+        EdmApi edmApi = retrofit.create( EdmApi.class );
+        PermissionsApi permissionApi = retrofit.create( PermissionsApi.class );
+
+        Dataset<Row> health = sparkSession
+                .read()
+                .format("com.databricks.spark.csv")
+                .option("header", "true")
+                .load(healthPath);
+
+        RequiredEdmElements requiredEdmElements = ConfigurationService.StaticLoader
```

Comment:
We agreed to no longer use `edm.yaml` for integrations. Please remove the relevant code.